src/components/share/modal/ShareModal.tsx

```diff
@@ -132,12 +160,12 @@ export const ShareModal = <UserType, InvitationType, AccessType>({
 
     if (search === "") {
       setSearchQuery("");
-      props.onSearchUsers("");
+      props.onSearchUsers!("");
```

Comment:
Maybe i don't understand but...

```suggestion
props.onSearchUsers?.("")
```